symex-interface-clojure.el

```diff
@@ -75,6 +75,20 @@ Accounts for different point location in evil vs Emacs mode."
   "Evaluate buffer."
   (cider-eval-buffer))
 
+(defun symex-clojure-comment-remaining-reader-macro ()
+  "Comment out remaining symexes at this level using clojure's comment reader macro."
+  (interactive)
+  (let ((count (symex--remaining-length)))
+    (symex-clojure-comment-reader-macro count)))
+
+(defun symex-clojure-comment-reader-macro (count)
+  "Comment symex using clojure's comment reader macro"
```

Comment:
I was a little confused by this docstring - maybe it should say "... using clojure's syntax for commenting reader macros"